lib/archethic/oracle_chain/services/uco_price/providers/coin_marketcap.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Archethic.OracleChain.Services.UCOPrice.Providers.CoinMarketCap do
+  @moduledoc false
+
+  alias Archethic.OracleChain.Services.UCOPrice.Providers.Impl
+
+  @behaviour Impl
+
+  require Logger
+
+  @impl Impl
+  @spec fetch(list(binary())) :: {:ok, %{required(String.t()) => any()}} | {:error, any()}
+  def fetch(pairs) when is_list(pairs) do
+    query = 'http://coinmrketcap.com/currencies/uniris/markets/'
```

Comment:
The link is not the good one, there is a typo error. Also you should use https request instead of http. In the httpc.request function you should use the ssl verification as it is done for coingecko provider

Reply:
Hi Neylix. Indeed, sorry, yesterday while testing I introduced this typo to test resilience of agregator when service crash. I will revert to correct address.